config/environments/development.rb

```diff
@@ -33,7 +33,7 @@
 
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
-
+  config.action_mailer.delivery_method = :file
```

Comment:
Just in case can we add this to `profiling.rb` as well?  It inherits from production normally which would default to `ses`.